src/app/models/AssociationDecorators.ts

```diff
@@ -45,7 +45,7 @@ export function Deleter<Parent extends AbstractModel & { deleterId?: Id }>() {
  *
  * @param serviceToken Injection token for API service used to retrieve the child models
  * @param identifierKeys Parent model key used to retrieve the list of ids for the child models
- * @param childIdentifier Primary key of the child model
+ * @param childIdentifier field used to filter child models by, typically ìd`
```

Comment:
Looks like by suggestion was bad: I has the international keyboard on!
```suggestion
 * @param childIdentifier field used to filter child models by, typically `id`
```